modules/lite/test/index.js

```diff
@@ -41,7 +41,7 @@ const nonGeoExample = new deck.DeckGL({
   container: document.getElementById('non-geo'),
   mapbox: false /* disable map */,
   views: [new deck.OrbitView()],
-  viewState: {distance: 1, rotationX: 45, rotationOrbit: 30, zoom: 0.05},
+  viewState: {distance: 1, fov: 50, rotationX: 45, rotationOrbit: 30, zoom: 0.05},
```

Comment:
* Ideally projection parameters like `fov` that are not really related to the viewState should go into the `OrbitView` itself. Not 1009% sure if that works for `OrbitView`.

* Any reason to change `viewState` to `initialViewState`? Do any docs need updating in this regard?

Reply:
`initialViewState` is needed to enable auto control